bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/conditions/LicensingCondition.java

```diff
@@ -49,12 +63,17 @@
 	Date getValidUntil();
 
 	/**
-	 * the type of condition like "time" or "hardware"
-	 * 
-	 * @return
+	 * The type of condition like "time" or "hardware".
+	 *
+	 * @return condition type
 	 */
 	String getConditionType();
 
+	/**
+     * Returns additional licensing requirements encoded in a single string value.
+     *
+     * @return description of additional demands of this licencing condition
+     * */
```

Comment:
Actually this not a requirement, but an expression like `hdd.serial=AAAAXXXX` to evaluate with a `PermissionEmitter` bound to a `conditionType`